### DISCUSSION TITLE
More facebook app event  with Subscribe and StartTrial events

### DISCUSSION THREAD

**Discussion by yelkamel** [oddbit/flutter_facebook_app_events#119] [Ideas]
Hello awesome package to use facebook analytics thanks you. 

Can you help me ? I need to use facebook app event  with Subscribe and StartTrial events.
https://developers.facebook.com/docs/marketing-api/conversions-api/subscription-lifecycle-events

Regards,

Youcef EL KAMEL

**Comment by DennisAlund** [comment]
Thanks for your input, we're trying to cover all the supported standard events. Sorry for the delay. In the meanwhile, you can still log those events by creating some extension methods that wraps and use custom logger. The event name is stated here: https://github.com/facebook/facebook-android-sdk/blob/b8cc91b4583f6b021cc06272c3f153a1736772eb/facebook-core/src/main/java/com/facebook/appevents/AppEventsConstants.kt#L117-L118

And see this discussion for a previous case where I didn't have time yet to implement the methods. Hope that helps as a workaround while I find the time 🙏 

https://github.com/oddbit/flutter_facebook_app_events/pull/67#issuecomment-784637262

**Reply by DennisAlund** [reply]
Fixed in `0.13` (see commit 7799a4dfd9caaef4c41a87eecbb525bcce0f1249)
